### DISCUSSION TITLE
[117.0 UPDATE FAIL] - Impossible to update Immich from 1.116.2 to 1.117.0

### DISCUSSION THREAD

**Discussion by nono2356** [immich-app/immich#13313] [Q&A] [answered]
Hi community,

I have a fail upgrade when I try to upgrade to 117.0. Immich runs with Docker Compose through an Unraid server.

Here is the error when I try to update, buy maybe you need more logs to help me ? Let me know and I will give it
![image](https://github.com/user-attachments/assets/b3153a9d-f574-4566-8465-5dd62af2b8b5)

Thanks for you help

**Comment by bo0tzz** [comment]
Your disk is full.

**Reply by nono2356** [reply, marked as answer]
Thanks so much @bo0tzz 
Ok (I upgraded storage for Docker Compose)
